storage_media/wizards/replace_file.xml

```diff
@@ -8,8 +8,8 @@
         <field name="arch" type="xml">
             <form string="Replace File" version="7.0">
                 <sheet>
-                    <field name="file_id" invisible="1" />
-                    <field name="media_id" invisible="1" />
+                    <field name="media_id" invisible="True" />
+                    <field name="file_id" invisible="True" />
```

Comment:
Should they be removed?

Reply:
Hehe, you've realized the interesting point of this. It will be the point to get data from the `default_get` function from wizzard.

You can try with `test_wizard_change_file` if the above 2 fields are not available.

Reply:
So as we experimented, looks it's not useful, we either:

1. Leave it as it is and hopefully nobody notices it :)
2. Remove it and wait for the author's reply. Pros: less code or even learn a new technique :))

It's your call :>